.github/workflows/quest-bulk.yml

```diff
@@ -25,25 +29,20 @@ jobs:
           echo "Reason: ${{ github.event.inputs.reason }}"
 
       - name: Azure OpenID Connect
-        uses: azure/login@v2
+        id: azure-oidc-auth
+        uses: dotnet/docs-tools/.github/actions/oidc-auth-flow@main
         with:
           client-id: ${{ secrets.CLIENT_ID }}
           tenant-id: ${{ secrets.TENANT_ID }}
           audience: ${{ secrets.OSMP_API_AUDIENCE }}
-          allow-no-subscriptions: true
-
-      - name: OSMP API access
-        run: |
-          TOKEN=$(az account get-access-token --query 'accessToken' -o tsv --resource ${{ secrets.OSMP_API_AUDIENCE }})
-          echo "AZURE_ACCESS_TOKEN=$TOKEN" >> $GITHUB_ENV
 
       - name: bulk-sequester
         id: bulk-sequester
         uses: dotnet/docs-tools/actions/sequester@main
         env:
           ImportOptions__ApiKeys__GitHubToken: ${{ secrets.GITHUB_TOKEN }}
           ImportOptions__ApiKeys__QuestKey: ${{ secrets.QUEST_KEY }}
-          ImportOptions__ApiKeys__AzureAccessToken: ${{ env.AZURE_ACCESS_TOKEN }}
+          ImportOptions__ApiKeys__AzureAccessToken: ${{ steps.azure-oidc-auth.outputs.access-token }}
           ImportOptions__ApiKeys__SequesterPrivateKey: ${{ secrets.SEQUESTER_PRIVATEKEY }}
           ImportOptions__ApiKeys__SequesterAppID: ${{ secrets.SEQUESTER_APPID }}
         with:
```

Comment:
There was a mistake in the source docs PR that we fixed yesterday. You need to add this line after line 52:

```
duration: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.duration || 5 }}
```

See https://github.com/dotnet/docs/blob/main/.github/workflows/quest-bulk.yml#L51C11-L51C105